lib/Controller/RoomController.php

```diff
@@ -796,9 +796,10 @@ protected function createOneToOneRoom(string $targetUserId): DataResponse {
 			// Actually createOneToOneConversation also checks it.
 			$room = $this->manager->getOne2OneRoom($currentUser->getUID(), $targetUser->getUID());
 			$this->participantService->ensureOneToOneRoomIsFilled($room, $currentUser->getUID());
+			$thread = $this->threadService->findForLastMessageByRoom($room);
```

Comment:
we should delay this deep into the RoomFormatter where the last message is also formated and then cache the needed info of the Message::toArray method only.